_contentTemplates/common/observable-data.md

```diff
@@ -0,0 +1,25 @@
+#intro
+The most common reason you would use an ObservableCollection is to make a component (like a grid, treeview, treelist, dropdown) change or react when you change that collection.
+
+When you want to refresh the component data source like that, there are two important framework behaviors you need to be aware of.
```

Comment:
Consider adding those behaviors here, something like

When you want to refresh the component data source like that, there are two important framework behaviors you need to be aware of - when ObservableCollection instances fire events, and how to refresh the data of a component when it is not an observable collection.

Reply:
Added.